diff --git a/python/BioSimSpace/Process/_gromacs.py b/python/BioSimSpace/Process/_gromacs.py
index 60dcfb0..406944f 100644
--- a/python/BioSimSpace/Process/_gromacs.py
+++ b/python/BioSimSpace/Process/_gromacs.py
@@ -781,6 +781,8 @@ class Gromacs(_process.Process):
         self.setArg("mdrun", True)          # Use mdrun.
         self.setArg("-v", True)             # Verbose output.
         self.setArg("-deffnm", self._name)  # Output file prefix.
+        self.setArg("-ntmpi", 1)            # One MPI thread.
+        self.setArg("-ntomp", 2)            # Two OpenMP threads per MPI thread.
 
         # Metadynamics and steered MD arguments.
         if type(self._protocol) is _Protocol.Metadynamics or \
