diff --git a/python/BioSimSpace/Metadynamics/_metadynamics.py b/python/BioSimSpace/Metadynamics/_metadynamics.py
index 3d429fb..645d349 100644
--- a/python/BioSimSpace/Metadynamics/_metadynamics.py
+++ b/python/BioSimSpace/Metadynamics/_metadynamics.py
@@ -139,17 +139,17 @@ def run(system, protocol, engine="auto", gpu_support=False, auto_start=True,
 
     for engine, exe in zip(engines, exes):
         try:
-            # AMBER.
-            if engine == "AMBER":
-                process = _Process.Amber(system, protocol, exe=exe, name=name,
-                    work_dir=work_dir, seed=seed, property_map=property_map)
-
             # GROMACS.
-            elif engine == "GROMACS":
+            if engine == "GROMACS":
                 process = _Process.Gromacs(system, protocol, exe=exe, name=name,
                     work_dir=work_dir, seed=seed, property_map=property_map,
                     ignore_warnings=ignore_warnings, show_errors=show_errors)
 
+            # AMBER.
+            elif engine == "AMBER":
+                process = _Process.Amber(system, protocol, exe=exe, name=name,
+                    work_dir=work_dir, seed=seed, property_map=property_map)
+
             # OPENMM.
             elif engine == "OPENMM":
                 if gpu_support:
